src/core/ui/upload/FormikVisualUpload/FormikVisualUpload.tsx

```diff
@@ -31,16 +31,17 @@ const FormikAvatarUploadSkeleton = ({ height = DEFAULT_SIZE, ...containerProps }
   );
 };
 
-interface VisualWithAltText {
+export type VisualWithAltText = {
   file: File;
   altText?: string;
-}
+};
 
 export interface FormikAvatarUploadProps extends BoxProps {
   name: string;
   visualType: VisualType;
   height?: number;
   altText?: string;
+  getNewAvatar?: (avatar: VisualWithAltText) => void;
```

Comment:
what I see here is that you are using this as a callback, is not a get or set.. I'd call it `onChange` or `onChangeAvatar` no?